getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,50 @@
+# Instructions for macOS
+
+Install required software:
+- [Homebrew](http://brew.sh/)
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+- [Vagrant](https://www.vagrantup.com/downloads.html)
+
+Install some basic command line tools, ansible and nodejs:
+
+```shell
+$ brew install ansible coreutils node
```

Comment:
This doesn't work with `brew`. Much to my chagrin, you actually need to have one package per command.